frontend/tests/features/wheresMyGene/tissueAutoExpand.test.ts

```diff
@@ -104,9 +135,184 @@ describe("WMG tissue auto-expand", () => {
+      EXPECTED_FILTERED_TISSUES_WITH_SEX_FILTER.length
+    );
+    await checkTissues(
+      page,
+      EXPECTED_FILTERED_TISSUES_WITH_SEX_FILTER,
+      EXPECTED_EXPANDED_TISSUES
+    );
+  });
+
+  /**
+   * Tissue auto expansion - cross filter with Publication filter selected
+   * Filter 'Ahren et al. Cell 2022' from the Publication filter. Filter Blood
+   * from the tissue filter. Blood should be expanded. Collapse Blood. add Cell
+   * filter for B Cell. Only B Cell should appear and Blood should be expanded.
+   */
+  test("Tissue auto expansion - cross filter with Publication filter selected", async ({
+    page,
+  }) => {
+    const tissues = ["blood"];
+    await loadPageAndTissues(page);
+    await filterSelection(
+      page,
+      PUBLICATION_FILTER_TEST_ID,
+      PUBLICATION_FILTER_LABEL,
+      PUBLICATION_FILTER_SELECTION[0]
+    );
+    await filterTissues(page, tissues);
+    await collapseTissue(page, tissues[0]);
+    await filterCellType(page, 1);
+    await checkTissues(page, tissues);
+    await checkElementVisible(page, EXPECTED_VISIBLE_CELL, CELL_TYPE_TEST_ID);
+  });
+
+  /**
+   * Tissue auto expansion - cross filter with Publication filter, check expansion
+   * Filter 3 Tissues ["abdomen", "axilla", "blood"]. Collapse Blood. Filter
+   * 'Ahren et al. Neuron 2021' from the Publication filter. Only Blood should remain
+   * visible and collapsed. Add Cell filter for B Cell. Only B Cell should be
+   * visible under expanded Blood tissue node
+   */
+  test("Tissue auto expansion - cross filter with Publication filter, check expansion", async ({
+    page,
+  }) => {
+    await loadPageAndTissues(page);
+    await filterTissues(page);
+    await collapseTissue(page, "blood");
+    await filterSelection(
+      page,
+      PUBLICATION_FILTER_TEST_ID,
+      PUBLICATION_FILTER_LABEL,
+      PUBLICATION_FILTER_SELECTION[1]
+    );
+    await expect(page.getByTestId(TISSUE_NODE_TEST_ID)).toHaveCount(1);
+    await checkTissues(page, ["blood"], []);
+  });
+
+  /**
+   * Tissue auto expansion - cross filter with Self-Reported Ethnicity filter
+   * Expand Breast. Select African from Ethnicity filter. Breast should remain expanded.
+   * Add Nose to the Tissue filter. Nose should appear expanded.
+   */
+  test("Tissue auto expansion - cross filter with Self-Reported Ethnicity filter", async ({
+    page,
+  }) => {
+    await loadPageAndTissues(page);
+    await expandTissue(page, SELF_REPORTED_ETHNICITY_TISSUE[0]);
+    await filterSelection(
+      page,
+      SELF_REPORTED_ETHNICITY_FILTER_TEST_ID,
+      SELF_REPORTED_ETHNICITY_FILTER_LABEL,
+      SELF_REPORTED_ETHNICITY_FILTER_SELECTION
+    );
+    await filterTissues(page, SELF_REPORTED_ETHNICITY_TISSUE);
+    await checkTissues(page, SELF_REPORTED_ETHNICITY_TISSUE);
+  });
+
+  /**
+   * Tissue auto expansion - cross filter with Disease filter
+   * Filter 3 Tissues ["abdomen", "axilla", "brain"]. From the Disease filter select influenza.
```

Comment:
seems like we're filtering just "blood"! Not sure if the test comment needs to be updated or the code 😄

Reply:
https://github.com/chanzuckerberg/single-cell-data-portal/pull/5924/files#r1349230230